espanol/transcender/transcender.md

```diff
@@ -5,4 +5,6 @@ su existencia el Consejero. Una de las últimas figuras que, por así decir, des
 ya no era mas que una masa de colores amorfa que aún se diferenciaba del "fondo".
 
 Pero aún persistian sus pensamientos. Debería desprenderse de esos también, si quería transcender 
-debidamente.
\ No newline at end of file
+debidamente. 
+
+Y lo hubiera hecho... Sin embargo, (una hora antes había sellado su propio destino)[destino.md].
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.